airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/DatabaseConfigPersistence.java

```diff
@@ -90,6 +90,11 @@ public void loadData(final ConfigPersistence seedConfigPersistence) throws IOExc
     });
   }
 
+  @Override
+  public Set<String> getConnectorReposInUse() throws IOException {
+    return database.transaction(this::getConnectorRepositoriesInUse);
```

Comment:
can we rename `getConnectorRepositoriesInUse` => `getInUseConnectorDockerImageNames`? I had to go read the impl to understand what that method was doing.

Reply:
Good suggestion, done!